spec/features/editing_file_attachments/edit_file_attachment_file_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.feature "Edit a file attachment file", js: true do
```

Comment:
JS isn't strictly needed for this feature, is it? Omitting `js: true` should make it run faster.

Reply:
Ah, good spot! I've added a commit to remove it.